spec/acceptance/suites/stunnel/03_stunnel_test_spec.rb

```diff
@@ -38,7 +38,7 @@
 ---
 # Need to test that our iptables works properly with this stuff
 
-use_iptables : true
+firewall : true
```

Comment:
In the acceptance tests, why isn't the hieradata using simp_options, e.g.
 simp_options::trusted_nets
 simp_options::firewall
 simp_options::auditd